### PR TITLE
Add kebechet schema

### PR DESCRIPTION
**Pull request by hemajv**
## Related Issues and Dependencies

#24

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

Adding generic schema for Kebechet

### CODE REVIEW

File: examples/kebechet/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+> I'm Kebechet, goddess of freshness. I will keep your sources and dependencies fresh and up-to-date.
+
+[Kebechet](https://github.com/thoth-station/kebechet) is a SourceOps bot that automates updating dependencies of your project. Currently, it supports managing and
+updating Python projects based on [pipenv](https://docs.pipenv.org/) files (`Pipfile` and `Pipfile.lock`) or
+`requirements.txt`/`requirements.in` files (see [pip-tools](https://pypi.org/project/pip-tools/) - Kebechet is a
+replacement for it).
+
+## Architecture
+
+The components and workflow of Kebechet can be found in the architecture schema below:
```
Comment (pacospace) [inline examples/kebechet/README.md:12]: `workflows` maybe, we will be more detailed in each of them

**Review by pacospace — COMMENTED** [review]
Thanks!! One minor component!

File: examples/kebechet/README.md (hunk 1 above)
Comment (hemajv) [reply examples/kebechet/README.md]: updated :+1:

**Review by pacospace — APPROVED** [review]
Thanks!!! 💯

**Pull request merged**